AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginFunctionalTests/GraphQLConnectionScenario3Tests+Subscribe.swift

```diff
@@ -92,114 +91,95 @@ extension GraphQLConnectionScenario3Tests {
         _ = try await Amplify.API.mutate(request: .create(post))
         _ = try await Amplify.API.mutate(request: .update(post))
 
-        try await AsyncExpectation.waitForExpectations([progressInvoked], timeout: TestCommonConstants.networkTimeout)
+        await waitForExpectations([progressInvoked], timeout: TestCommonConstants.networkTimeout)
 
         await task.cancel()
```

Comment:
Seeing warning message : `No 'async' operations occur within 'await' expression`

Reply:
latest PR to add API.subscribe removes this https://github.com/aws-amplify/amplify-ios/pull/2236